web/src/user_search.ts

```diff
@@ -115,7 +115,15 @@ export class UserSearch {
     initiate_search(): void {
         this.expand_column();
         this.show_widget();
-        this.$input.trigger("focus");
+        // For node tests to pass.
+        // eslint-disable-next-line no-undef
+        if (process.env.NODE_ENV === "test") {
+            this.$input.trigger("focus");
+        }
+        // Needs to be called when input is visible after fix_invite_user_button_flicker.
+        setTimeout(() => {
+            this.$input.trigger("focus");
+        }, 0);
```

Comment:
Not sure how to better handle this is node tests. The setTimeout call seems to trigger after the node tests finish.

Reply:
```
    set_global("setTimeout", (func) => {
        if (set_timeout_function_called) {
            // This conditional is needed to avoid indefinite calls.
            return;
        }
        set_timeout_function_called = true;
        func();
    });
```

We use this pattern in `activity_ui.test.js`.

Reply:
oh yeah! Updated!